packit_service/worker/helpers/sync_release/sync_release.py

```diff
@@ -7,7 +7,11 @@
 from ogr.abstract import GitProject
 
 from packit.config import JobType, PackageConfig, JobConfig, JobConfigTriggerType
-from packit.config.aliases import get_branches
+from packit.config.aliases import (
+    get_branches,
+    get_ff_branches_from as _get_ff_branches_from,
```

Comment:
Why the rename?

Reply:
Because there is a `get_ff_branches_from` in packit-service in this same module that uses the `get_ff_branches_from` taken from packit. I don't know what is better if the renaming or change completely the name in packit service. I will follow your suggestion.

Reply:
If I'm not missing anything there is only `SyncReleaseHelper.get_ff_branches_from()` method in this module, so there is no name conflict.

Reply:
> If I'm not missing anything there is only `SyncReleaseHelper.get_ff_branches_from()` method in this module, so there is no name conflict.

There is `SyncReleaseHelper.get_ff_branches_from()` which is using `_get_ff_branches_from` renamed from `get_ff_branches_from` taken from `packit.aliases`.